aspdotnet/datadog_checks/aspdotnet/aspdotnet.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under Simplified BSD License (see LICENSE)
 try:
     from datadog_checks.base import PDHBaseCheck
+
```

Comment:
Did this get missed?

Reply:
The decorator is useless here, as we'll trace every method of `AspdotnetCheck` excluding the constructor, which is none